mbed_app.json

```diff
@@ -20,11 +20,17 @@
         "password": {
             "help": "The password string to use for this APN, set to 0 if none",
             "value": 0
+        },
+        "trace-level": {
+            "help": "Options are TRACE_LEVEL_ERROR,TRACE_LEVEL_WARN,TRACE_LEVEL_INFO,TRACE_LEVEL_DEBUG",
+            "macro_name": "MBED_TRACE_MAX_LEVEL",
+            "value": "TRACE_LEVEL_INFO"
         }
-	},
+  },
     "target_overrides": {
         "*": {
             "target.features_add": ["LWIP", "COMMON_PAL"],
+            "mbed-trace.enable": false,
```

Comment:
Should we use here by default "platform.stdio-buffered-serial": true

Reply:
Yes, fixed.